doc/Theming.md

```diff
@@ -181,3 +181,21 @@ entity.errorMessage(function (response) {
     return 'Global error: ' + response.status + '(' + response.data + ')';
 });
 ```
+## Customizing HTTP Error Messages
+If you want to override, patch or extend the way HTTP error are handled by the ng-admin Http Error Service (../src/javascripts/ng-admin/Main/component/provider/HttpErrorService.js) then you may use an [Angular decorator](https://docs.angularjs.org/guide/decorators), as per the below example:
```

Comment:
Can you fix the link on `Http Error Service`? 

@Kmaschta: seems sedy failed. Moreover, would it work if I try to fix the link (with slashes in it)?